docs/moduleCom1DFA.rst

```diff
@@ -63,7 +63,9 @@ projection:
   - if you want to simulate different scenarios with the same features, you have to copy them to separate shapefiles
 
 
-and the following files are optional:
+and the following files are optional. Please note: in the standard configuration, the *null* variant is always run! I
+.e. if some of the following files are given, at least two results are generated: the *null* variant and the variant
+with entrainment and/or resistance.
```

Comment:
consider adding and secondary release areas are always triggered as soon as any part of the secondary release are is affected by the snow avalanche flow